packages/client/src/containers/LandingPage/LandingPage.Container.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Testimonials from '../../components/Login/Testimonials';
+import logo from './logo.svg';
+import { Typography, TextField, Button } from '@mui/material';
+import './LandingPage.Style.css';
+
+const LandingPageContainer = () => {
+  return (
+    <div className="landing-page">
+      <div className="testimonial-container">
+        <Typography className="text-field" variant="h4" color="white">
+          Don’t forget to invite teammates!
+        </Typography>
+        <div className="testimonial-cards">
+          <Testimonials />
+        </div>
+      </div>
+
+      <div className="login-container">
+        <img className="logo" src={logo} alt="Logo" />
+        <div className="login-details">
+          <Typography variant="h5" fontWeight="bold">
```

Comment:
Same here with the `fontWeight`, this could be done in css.

Reply:
Same as above ☝🏽

Reply:
@shpomp We at Workday struggle a lot with custom CSS, and our aim is to use as less CSS as possible. But if you need to change a styling in an unknown file/feature it would be easy to assume that if you need to change a styling issue it should be located in the `css` file. It is all coming down to a common understanding, and these styling changes should be either built in the `<Typography.H5>` to have the same style everywhere, or if we are overwriting it locally it should be coming from the `.css` file.
I hope it make sense what I blabbered 😅